submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/model/SubmissionRequest.kt

```diff
@@ -18,16 +18,22 @@ data class SubmissionRequest(
     val notifyTo: String,
     val status: RequestStatus,
     val totalFiles: Int,
+    val conflictedFiles: Int,
+    val deprecatedFiles: Int,
     val currentIndex: Int,
     val modificationTime: OffsetDateTime,
+    val previousVersion: Int?,
```

Comment:
I just realized we have the `version` field in the `DocSubmissionRequest` class but we don't map it here, I'm not sure why but I think we should add it

Also, I noticed the class `RequestStatusChanges` (just above this one) appears as unused. Maybe we should delete it or is it intended to be used in a later PR?

Reply:
we do not include the version here because is kind of in the submission object. I mean version =  submission.version. Still if you think is there any use for it a root level I can added it but is redudant.